src/actions/reviewActions.js

```diff
@@ -0,0 +1,54 @@
+import {
+  FETCH_REVIEWS,
+  FETCH_SINGLE_REVIEW,
+  POST_REVIEW,
+  EDIT_REVIEW,
+  DELETE_REVIEW
+} from "./actionTypes";
+
+import axios from "axios";
+
+export const listReviews = () => async dispatch => {
+  try {
+    const response = await axios.get("/api/reviews");
+    dispatch({ type: FETCH_REVIEWS, payload: response.data });
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+export const singleReviewThunk = id => async dispatch => {
```

Comment:
Naming conventions are key - if you are post-fixing thunk to var names that are thunks do it everywhere!